core/src/main/scala/org/apache/spark/sql/delta/DeltaOptions.scala

```diff
@@ -163,6 +163,9 @@ trait DeltaReadOptions extends DeltaOptionParser {
 
   val ignoreDeletes = options.get(IGNORE_DELETES_OPTION).exists(toBoolean(_, IGNORE_DELETES_OPTION))
 
+  val skipChangeCommits = options.get(SKIP_CHANGE_COMMITS_OPTION)
```

Comment:
Can you add a comment explaining this variable and what it means? e.g. how does it relate to ignoreDeletes and ignoreChanges?

Reply:
Updated the description to include more info